vocabs/v1/context.jsonld

```diff
@@ -1,89 +1,59 @@
 {
   "@context": {
-    "@version": 1.1,
+    "@protected": true,
     "id": "@id",
     "type": "@type",
-    "dc": "http://purl.org/dc/terms/",
-    "schema": "http://schema.org/",
-    "sec": "https://w3id.org/security#",
-    "didns": "https://www.w3.org/ns/did#",
-    "xsd": "http://www.w3.org/2001/XMLSchema#",
-    "EcdsaSecp256k1VerificationKey2019": "sec:EcdsaSecp256k1VerificationKey2019",
-    "Ed25519Signature2018": "sec:Ed25519Signature2018",
-    "Ed25519VerificationKey2018": "sec:Ed25519VerificationKey2018",
-    "JsonWebKey2020": "sec:JsonWebKey2020",
-    "JsonWebSignature2020": "sec:JsonWebSignature2020",
-    "Bls12381G1Key2020": "sec:Bls12381G1Key2020",
-    "Bls12381G2Key2020": "sec:Bls12381G2Key2020",
-    "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
-    "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
-    "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
-    "ServiceEndpointProxyService": "didns:ServiceEndpointProxyService",
-    "LinkedDomains": "https://identity.foundation/.well-known/resources/did-configuration/#LinkedDomains",
+
     "alsoKnownAs": {
       "@id": "https://www.w3.org/ns/activitystreams#alsoKnownAs",
       "@type": "@id",
       "@container": "@set"
     },
     "assertionMethod": {
-      "@id": "sec:assertionMethod",
+      "@id": "https://w3id.org/security#assertionMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "authentication": {
-      "@id": "sec:authenticationMethod",
+      "@id": "https://w3id.org/security#authenticationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityDelegation": {
-      "@id": "sec:capabilityDelegationMethod",
+      "@id": "https://w3id.org/security#capabilityDelegationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityInvocation": {
-      "@id": "sec:capabilityInvocationMethod",
+      "@id": "https://w3id.org/security#capabilityInvocationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "controller": {
-      "@id": "sec:controller",
+      "@id": "https://w3id.org/security#controller",
       "@type": "@id"
     },
-    "created": {
-      "@id": "dc:created",
-      "@type": "xsd:dateTime"
-    },
-    "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
-      "@id": "sec:keyAgreementMethod",
+      "@id": "https://w3id.org/security#keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKey": {
-      "@id": "sec:publicKey",
-      "@type": "@id",
-      "@container": "@set"
-    },
-    "publicKeyBase58": "sec:publicKeyBase58",
```

Comment:
I think removing these are correct so that we don't cause conflict with the LDP context in the future, but we're also defining them within the spec in. Would it make more sense to leave this in here because of that or can we get away with removing this and also using the security context in JSON-LD DID Documents?

Reply:
The current proposal is for JSON-LD DID Documents to explicitly pull in the cryptosuites that they support. So, for example, a did:key that supports ed25519 would do this:

```
  "@context": [
    "https://www.w3.org/ns/did/v1",
    "https://w3id.org/security/suites/ed25519-2020/v1"   // NOTE: Non-existent right now, but will exist within a week or two
  ],
```

DID Methods could also choose to do this:

```
  "@context": [
    "https://www.w3.org/ns/did/v1",
    "https://w3id.org/example-did-method/v1"
  ],
```

The rationale being that dumping every modern cryptosuite into the did/v1 context has downsides... it's potential attack surface. So rather than kitchen-sink the did/v1 context, the suggestion is to make the cryptosuites the DID Document supports explicit.

So, to answer your question, things like `publicKeyBase58` will be defined by a cryptosuite specification, which can be included by adding the cryptosuite context file to `@context`, or by subsuming it into a broader DID Method context, which is then included in `@context`.

Reply:
Ahh good point, makes sense and thanks for clarifying that nuance.